### PR TITLE
2020 schedule

### PR DESCRIPTION
**Pull request by garrying**

### CODE REVIEW

**Review by dcwalk — APPROVED** [review]
Looks good!

File: _data/sessions.yml [hunk 1]
```diff
@@ -1405,5 +1406,42 @@
...
+- sessionID: 20040
+  title: "Opening remarks"
+  sessionType: "orga"
+
+- sessionID: 20041
+  title: "Closing remarks"
+  sessionType: "orga"
+
+- sessionID: 20046
+  title: "Gather.town"
```
Comment (dcwalk) [inline _data/sessions.yml:1448]: Can we mark these as a "hangout"?

File: program/exhibited.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+layout: page
+order: 3
+title: "Exhibited works"
```
Comment (dcwalk) [inline program/exhibited.md:4]: How are we going to change the behaviour to link in to these? When should we link down to program vs. directly to site? When should that go live?

Comment (garrying) [reply program/exhibited.md:4]: I think there could be a CTA for visiting the site in both the schedule and program. Revisiting this in another PR.

File: _data/sessions.yml (hunk 1 above)
Comment (garrying) [reply _data/sessions.yml:1448]: updated in a723e46

**Pull request merged**